### PR TITLE
Fix: Correct module path and clarify config structure in Medusa v2 Docs

### LINKED ISSUE #9220: Medusa v2 Docs: Typo in module creation example + clarification
### What Medusa version and documentation are you using?

v2

### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/medusajs/medusa/issues

### Issue Summary

In https://docs.medusajs.com/v2/basics/modules-and-services#test-the-module, the example provided contains

```typescript
import HelloModuleService from "../../../modules/hello/service"
import { HELLO_MODULE } from "../../../modules/hello"
```

It should only go two levels up:
```typescript
import HelloModuleService from "../../modules/hello/service"
import { HELLO_MODULE } from "../../modules/hello"
```

Also for clarity, in the example of `medusa-config.js` (https://docs.medusajs.com/v2/basics/modules-and-services#3-add-module-to-configurations), you probably want to clarify how the structure is altered. I myself made the mistake of putting the `modules` node under `projectConfig` and got stuck for a while. Current version in the documentation:
```typescript
module.exports = defineConfig({
  // ...
  modules: {
    helloModuleService: {
      resolve: "./modules/hello",
    },
  },
})
```

Suggested change to clarify and avoid other people doing the same mistake:
```typescript
module.exports = defineConfig({
  projectConfig: {
    // ...
  },
  modules: {
    helloModuleService: {
      resolve: "./modules/hello",
    },
  },
})
```

### How can this issue be resolved?

1. Suggesting to adjust as per described above

### Are you interested in working on this issue?

- [x] I would like to fix this issue

### PR DESCRIPTION
**Pull request by aqeebpathan**
### What
This PR addresses an issue in the Medusa v2 documentation regarding the module creation example. The import paths were corrected to reflect the proper directory structure, ensuring clarity for users.

Closes #9220 

### Why
The previous documentation contained incorrect import paths that could lead to confusion for users attempting to create and configure modules. Clarifying the structure aims to prevent similar issues for future users.

### How
- Updated the import statements in the documentation to go two levels up instead of three.
- Added clarification in the `medusa-config.js` example to indicate that the `modules` node should be a part of `projectConfig`.

### Testing
While documentation changes do not require formal testing, I reviewed the updated paths and descriptions against the current code structure to ensure accuracy and clarity.

### Note
This is my first contribution to an open source project, and I'm excited to contribute to the Medusa community!

### CODE REVIEW

**Review by damien-thiesson — APPROVED** [review]
LGTM

**Comment by olivermrbl** [comment]
Thanks for the contribution. Will let @shahednasser take a look at it!

File: www/apps/book/app/basics/modules-and-services/page.mdx [hunk 1]
```diff
@@ -74,7 +74,9 @@ In `medusa-config.js`, add a `modules` property and pass to it your custom modul
 
 ```js title="medusa-config.js" highlights={[["4", "helloModuleService", "The key of the main service to be registered in the Medusa container."]]}
```
Comment (shahednasser) [inline www/apps/book/app/basics/modules-and-services/page.mdx]:
```suggestion
```js title="medusa-config.js" highlights={[["7", "helloModuleService", "The key of the main service to be registered in the Medusa container."]]}
```

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thanks for your contribution, just need a small change.

**Review by shahednasser — APPROVED** [review]
LGTM thanks!

**Pull request merged**